extensions/cornerstone/src/init.js

```diff
@@ -38,16 +74,23 @@ export default function init({ servicesManager, configuration }) {
       defaultPosition: _getDefaultPosition(event.detail),
       content: ContextMenuMeasurements,
       contentProps: {
+        onGetMenuItems,
         eventData: event.detail,
-        onDelete: item => {
-          console.log('TBD: onDelete', item);
+        onDelete: (item) => {
+          const { tool: measurementData, toolType } = item.value;
+          measurementServiceSource.remove(measurementData.id);
```

Comment:
@dannyrb do we really need both of these to delete an annotation?

Reply:
We could wire the measurement_removed event and remove from the toolstate as well.

Reply:
Looks like @dannyrb did this in the other PR, so I'll wait for that to be merged and we can fix this after that

Reply:
@dannyrb @swederik i'll leave this pr as is for now, ping me after the mentioned merge